spec/features/multi_factor_authentication/mfa_cta_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+feature 'mfa cta banner' do
+	include DocAuthHelper
+	include SamlAuthHelper
+  
+
+	context 'multiple factor authentication feature is disabled' do
```

Comment:
surprising indentation here (it's tabs... :/)
```suggestion
  include DocAuthHelper
  include SamlAuthHelper

  context 'multiple factor authentication feature is disabled' do
```